addons/account/models/ir_module.py

```diff
@@ -35,7 +35,11 @@ def _compute_account_templates(self):
         for module in self:
             templates = {}
             if module.category_id == chart_category or module.name == 'account':
-                python_module = import_module(f"odoo.addons.{module.name}.models")
+                try:
+                    python_module = import_module(f"odoo.addons.{module.name}.models")
+                except ModuleNotFoundError:
+                    module.account_templates = False
+                    continue
```

Comment:
I find something like this more readable
```python
                try:
                    python_module = import_module(f"odoo.addons.{module.name}.models")
                except ModuleNotFoundError:
                    templates = {}
                else:
                    templates = {
                        fct._l10n_template[0]: {
                            'name': fct(ChartTemplate).get('name'),
                            'parent': fct(ChartTemplate).get('parent'),
                            'country': fct(ChartTemplate).get('country', ''),
                            'visible': fct(ChartTemplate).get('visible', True),
                            'module': module.name,
                        }
                        for _name, mdl in getmembers(python_module, template_module)
                        for _name, cls in getmembers(mdl, template_class)
                        for _name, fct in getmembers(cls, template_function)
                    }
```

Then you don't need to change anything in `addons/account/models/chart_template.py`

Reply:
oh yes, it is much neater indeed